README.md

```diff
@@ -1,4 +1,31 @@
-# Simple-Calculator-Web-app
+# Simple-Calculator-Web-app -- An Amazing Project
```

Comment:
I have a few modification suggestions. Let's keep the original main title: `# Simple-Calculator-Web-app`